core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -9564,4 +9564,21 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
         .withBigQuery()
         .ok(expectedBQSql);
   }
+
+  @Test public void testTimeOfDayFunction() {
+    final RelBuilder builder = relBuilder();
+    final RexNode formatTimestampRexNode2 = builder.call(SqlLibraryOperators.FORMAT_TIMESTAMP,
+        builder.literal("TIMEOFDAY"),builder.call(SqlLibraryOperators.CURRENT_TIMESTAMP));
```

Comment:
add space before builder.call(SqlLibraryOperators.CURRENT_TIMESTAMP)

Reply:
Handled.

Reply:
You have given 2 spaces
Give only one space